libc/startup/gpu/CMakeLists.txt

```diff
@@ -26,6 +26,16 @@ function(add_startup_object name)
     PROPERTIES
       OUTPUT_NAME ${name}.o
   )
+
+  # Make an executable target of relocatable bitcode for clang if needed.
+  if(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR)
+    add_executable(${fq_target_name}.exe $<TARGET_OBJECTS:${fq_target_name}>)
+    set_target_properties(${fq_target_name}.exe PROPERTIES
+      RUNTIME_OUTPUT_DIRECTORY ${LIBC_LIBRARY_DIR}
+      RUNTIME_OUTPUT_NAME ${name}.o)
+    target_link_options(${fq_target_name}.exe PRIVATE 
+                        "-flto" "-Wl,--lto-emit-llvm" "-march=sm_90")
```

Comment:
Do you want to hard-code the `-march` here?

Reply:
Yeah that's a hack, I can probably patch clang for it.

Reply:
It pretty much just needs to be there to make clang happy, I'll just leave it blank for now.